query-service-api/src/main/proto/request.proto

```diff
@@ -73,6 +74,12 @@ message ColumnIdentifier {
   string alias = 2;
 }
 
+message AttributeExpression {
+  string attributeId = 1;
```

Comment:
```suggestion
  string attribute_id = 1;
```

(snake_case is our platform standard; normally I'd say be consistent, but given the difficulty of changing these later I think we should use correct casing on all new things)

Reply:
Aren't we using camel case everywhere ?

Reply:
this file is breaking our coding standards enforced in every other repo (or most of them). snake_case is our standard for proto messages, we follow the default lint style from https://docs.buf.build/lint/rules#default .

Reply:
resolved